packages/teams-js/src/private/files.ts

```diff
@@ -296,6 +296,70 @@ export namespace files {
     extension: string;
   }
 
+  export enum CloudStorageProviderFileAction {
+    New = 'NEW',
+    Rename = 'RENAME',
+    Download = 'DOWNLOAD',
+    Upload = 'UPLOAD',
+    Delete = 'DELETE',
+    Logout = 'LOGOUT',
+  }
+
+  export interface I3PCloudStorageProviderRequest<T> {
+    content: T;
+  }
+
+  /**
```

Comment:
I don't think these comments get turned into documentation since they aren't exported, so @hidden shouldn't be necessary (and you have comments on the exported types using them below)

Reply:
Removed the hidden marker, have retained the comments for clarity.